packages/core/test/lib/tracing/trace.test.ts

```diff
@@ -80,6 +81,18 @@ describe('startSpan', () => {
       expect(ref.status).toEqual(isError ? 'internal_error' : undefined);
     });
 
+    it('creates & finishes span', async () => {
+      let _span: Span | undefined;
+      startSpan({ name: 'GET users/[id]' }, span => {
+        expect(span).toBeDefined();
+        expect(span?.endTimestamp).toBeUndefined();
+        _span = span;
+      });
+
+      expect(_span).toBeDefined();
+      expect(_span?.endTimestamp).toBeDefined();
```

Comment:
```suggestion
      expect(_span.endTimestamp).toBeDefined();
```

Reply:
sadly, TS still complains about this even if I check for defined-ness above 😬